testsuite/MDAnalysisTests/data/coordinates/test_incomplete_vel.gro

```diff
@@ -0,0 +1,8 @@
+Written by MDAnalysis
+    5
+    1MET     CA    1   0.000   0.100   0.200  0.0000  0.0100  0.0200
+    2ARG     CA    2   0.300   0.400   0.500
```

Comment:
I agree with @jbarnoud here. Although there's broken and broken: we should properly race an exception if some of the coordinates are missing (which can happen when I/O goes down in the middle of writing); you'd likely have the final box information also missing but just to be clear:

- read coordinates and velocities when available (correct format)
- read only coordinates if v are missing (but issue warning (logger/warn))
- raise exception if coordinates are missing